data/json/items/comestibles/carnivore.json

```diff
@@ -2075,5 +2075,127 @@
     "volume": "250 ml",
     "fun": -10,
     "flags": [ "NO_SALVAGE" ]
+  },
+  {
+    "id": "poultry",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": { "str": "chunk of poultry meat", "str_pl": "chunks of poultry meat" },
+    "description": "A chunk of raw meat from a lean bird.  You could eat it raw, but it carries a risk of bacterial infection.",
+    "weight": "160 g",
+    "calories": 240,
+    "//": "Scaled from 100g boneless chicken thigh according to USDA database",
```

Comment:
Is this 100g of meat also 250ml in volume since it's copied the volume of the existing meat?  Please confirm

Reply:
<https://www.aqua-calc.com/page/density-table/substance/beef-blank-steak-coma-and-blank-ns-blank-as-blank-to-blank-cooking-blank-method-coma-and-blank-lean-blank-and-blank-fat-blank-eaten-blank--op-boneless-coma-and-blank-cooked-cp->  
I'm seeing 1.04 g/ml for both beef and chicken.  So you should have 260g of poultry and our meat item should also be 260g?

Reply:
What exactly is our meat based on, as a reference?

Reply:
Hopefully somewhere in this PR is an answer.  This is the kind of detail we should have in comments on baseline items like meat.  https://github.com/CleverRaven/Cataclysm-DDA/pull/27289